grails-app/controllers/au/org/ala/specieslist/AdminController.groovy

```diff
@@ -79,7 +81,12 @@ class AdminController {
 
     // Access managed by AdminInterceptor
     def updateListsWithUserIds() {
-        Boolean successful = userDetailsService.updateSpeciesListUserDetails()
-        render(status: successful ? 200 : 404, text: "Update of editors was ${(successful)?'':'not'} successful")
+        def result = userDetailsService.updateSpeciesListUserDetails()
+        render result as JSON
+    }
+
+    def updateListsEditorWithIds() {
+        def result = userDetailsService.updateEditorsList()
```

Comment:
`userDetailsService.updateEditorsList()` does not exist

Reply:
The method updateListsEditorWithIds() in AdminController is intended to be removed.
Removed

Reply:
updateListsEditorWithIds() in AdminController is meant to be removed